Automated Smoketest Build: failed
Number of Error(s): 5
/mnt/disk1/home/vamosax/smoketest/smoketest-7876/HPCC-Platform/system/security/LdapSecurity/ldaputils.cpp:96:123: error: ‘LdapGetLastError’ was not declared in this scope
             throw MakeStringException(-1, ldap_initialize(%s,%d) error %s, host, port, ldap_err2string(LdapGetLastError()));
make[2]: *** [system/security/LdapSecurity/CMakeFiles/LdapSecurity.dir/ldaputils.cpp.o] Error 1
make[1]: *** [system/security/LdapSecurity/CMakeFiles/LdapSecurity.dir/all] Error 2
make: *** [all] Error 2

--- a/system/security/LdapSecurity/ldaputils.cpp
+++ b/system/security/LdapSecurity/ldaputils.cpp
@@ -93,7 +93,7 @@ LDAP* LdapUtils::LdapInit(const char* protocol, const char* host, int port, int 
         int rc = LDAP_INIT(&ld, uri.str());
         if(rc != LDAP_SUCCESS)
         {
-            throw MakeStringException(-1, "ldap_initialize(%s,%d) error %s", host, port, ldap_err2string(LdapGetLastError()));
+            throw MakeStringException(-1, "ldap_initialize(%s,%d) error %s", host, port, ldap_err2string(rc));
         }
 #endif
     }

